internal/providers/terraform/azure/registry.go

```diff
@@ -214,6 +216,9 @@ var FreeResources = []string{
 	"azurerm_firewall_policy",
 	"azurerm_firewall_policy_rule_collection_group",
 
+	// Azure Front Door
+	"azurerm_frontdoor_custom_https_configuration",
```

Comment:
Looks like Hashicorp have added support for a new resource `azurerm_frontdoor_rules_engine` since we broke this one down. I can't find any charges for it and https://www.neowin.net/news/azure-front-door-and-cdn-rules-engine-now-generally-available/ seems to suggest it's free, so I think we should add it here and to the docs.